esinstall/src/index.ts

```diff
@@ -317,7 +317,7 @@ ${colors.dim(
       }),
       rollupPluginCatchFetch(),
       rollupPluginNodeResolve({
-        mainFields: ['browser:module', 'module', 'browser', 'main'],
+        mainFields: MAIN_FIELDS,
```

Comment:
Can you change this to match ` [...packageLookupFields, ...MAIN_FIELDS]`? This is a smaller issue, but could still lead to bugs if you expected your lookup fields to apply to all packages.

Reply:
Good catch. Looks like a couple unrelated tests are failing, but I confirmed that they fail in the same way with & without this PR so I think these changes should be safe.